config/api_sample.php

```diff
@@ -24,6 +24,9 @@
         // When using unix socket to connect to the database the host attribute should be removed
         // 'socket' => '/var/lib/mysql/mysql.sock',
         'socket' => '',
+        'driver_options' => [
```

Comment:
Please comment these lines as they might cause issues with most people that are not using ssl

Reply:
done